apps/cli/src/__generated__/templates.ts

```diff
@@ -0,0 +1,84 @@
+/*
+
+This is a auto-generated file. Please don't change manually.
+If needed to make any changes. Add them to ./templates folder and run pnpm run build:templates
+
+*/
+
+import type { Folder, ProjectType } from "../utils";
+
+export const TEMPLATES: Record<ProjectType, Folder> = {
```

Comment:
personally i would go for 
```suggestion
export const templates: Record<ProjectType, Folder> = {
```

Reply:
@TrySound wdyt, whats our convention, its kinda weird because const is used a lot of times with camel case, I used to think when something is exported as a constant and is never changing, it should be all caps, but it might be also a bit annoying, so not sure here, I will go with whatever you and @istarkov think is best